src/components/pages/ViewAllMeetings/MeetingInfo.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const MeetingInfo = props => {
+  const { info } = props;
+
+  return (
+    <div>
+      <p>{info.meeting_id}</p>
+      <p>{info.created_at}</p>
+      <p>{info.updated_at}</p>
+      <p>{info.meeting_topic}</p>
+      <p>{info.meeting_time}</p>
+      <p>{info.host_id}</p>
+      <p>{info.attendee_id}</p>
+      <p>{info.meeting_notes}</p>
```

Comment:
i would say just show the names here of the host, which we already talked about, but aside from that everything looks good